content/asset-tracking/index.textile

```diff
@@ -342,3 +342,4 @@ h2(#see-also). See also
 * "Using the example apps":/asset-tracking/example-apps
 * "Using the SDKs":/asset-tracking/using-the-sdks
 * "Ably Asset Tracking Android SDK Reference - generated from source code":https://sdk.ably.com/builds/ably/ably-asset-tracking-android/main/dokka/index.html
+* "Reference documentation for Ably Asset Tracking SDKs for Swift":https://sdk.ably.com/builds/ably/ably-asset-tracking-swift/main/jazzy/
```

Comment:
Looks like you made your update on an old clone of the docs repo, resulting in a merge conflict. I've fixed the merge conflict, which basically updates the line above this one to include the fix me made on your previous related PR. 

We should also change the text for the Swift line to be consistent with the Android line.

Reply:
Done